phalcon/db/adapter/pdo.zep

```diff
@@ -283,12 +287,15 @@ abstract class Pdo extends Adapter
 		/**
 		 * Check for \PDO::XXX class constant aliases
 		 */
-        for key, value in options {
-            if typeof key == "string" && defined("\PDO::" . key->upper()) {
-                let options[constant("\PDO::" . key->upper())] = value;
-                unset options[key];
-            }
-        }
+		for key, value in options {
+			if typeof key == "string" {
+				let pdoKey = "\PDO::" . key->upper();
+				if defined(pdoKey) {
+					let options[constant(pdoKey)] = value;
+					unset options[key];
+				}
+			}
+		}
```

Comment:
Instead of trying to "guess" what the developer passed, why not leave it as simple `options` without trying to figure out what `\PDO` constant this is?

Reply:
@niden I believe that the class was already guessing in some way.  I'll have to get my head back into that but I tried to not disrupt it so much.  I'd be for a changing of the `options` API.  I also don't like the magic but the magic was already there.

Reply:
Since this is a big release and we _can break stuff_ so to speak, we can just have the options and let the developer pass whatever they want. I am also wondering if we even need to be checking for the `persistent` option since for PDO there is a particular option for it. 

Thoughts?